pkg/repository/v1/sqlcv1/olap.sql

```diff
@@ -4,7 +4,9 @@ SELECT
     create_v1_hash_partitions('v1_task_status_updates_tmp'::text, @partitions::int),
     create_v1_olap_partition_with_date_and_status('v1_tasks_olap'::text, @date::date),
     create_v1_olap_partition_with_date_and_status('v1_runs_olap'::text, @date::date),
-    create_v1_olap_partition_with_date_and_status('v1_dags_olap'::text, @date::date);
+    create_v1_olap_partition_with_date_and_status('v1_dags_olap'::text, @date::date),
+    create_v1_range_partition('v1_events_olap'::text, @date::date),
+    create_v1_range_partition('v1_event_to_run_olap'::text, @date::date);
 
 -- name: ListOLAPPartitionsBeforeDate :many
```

Comment:
We'll need to return event partitions before this date and make sure we detach old partitions from the events tables

Reply:
done in [this commit](https://github.com/hatchet-dev/hatchet/pull/1633/commits/da579ab280c120d5b3eb95dda89eda8a1ad3dd47) I think - is there anything else that'll need to be added for this? I grepped for `DETACH PARTITION` and this was what I found

Reply:
This looks correct but the best way to test it to create partitions older than the retention window and make sure that they get cleaned up on server startup

Reply:
copy that, will give it a shot

Reply:
okay yep, just confirmed that works!